### PR TITLE
apple: init MacBookPro11,1

### PR DESCRIPTION
**Pull request by j0hax**
###### Description of changes
Just a simple module I wrote for my late 2013 MacBook Pro and would like to share with the community.

###### Things done

- [x] Tested the changes in your own NixOS Configuration
- [x] Tested the changes end-to-end by using your fork of `nixos-hardware` and
      importing it via `<nixos-hardware>` or Flake input

### CODE REVIEW

File: apple/macbook-pro/11-1/default.nix [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+{
+  imports = [
+    ../.
+    ../../../common/pc/laptop/ssd
+    ../../../common/cpu/intel/haswell
+  ];
+
+  # broadcom-wl
+  hardware.enableRedistributableFirmware = lib.mkDefault true;
+  boot.extraModulePackages = with config.boot.kernelPackages; [ broadcom_sta ];
```
Comment (Mic92) [inline apple/macbook-pro/11-1/default.nix:11]: Is this driver not also picked up by nixos-generate-config?

Comment (j0hax) [reply apple/macbook-pro/11-1/default.nix:11]: It didn't seem to be the case when I installed NixOS last week: https://github.com/j0hax/nixos-configurations/blob/4533c6b77442014dbfe97d28e74d7caef07165b5/hosts/clay/hardware-configuration.nix

Comment (Mic92) [inline apple/macbook-pro/11-1/default.nix:11]:
```suggestion
  # nixos-generate-config doesn't detect this automatically.
  boot.extraModulePackages = with config.boot.kernelPackages; [ broadcom_sta ];
```

**Comment by Mic92** [comment]
@mergify queue

**Comment by Mic92** [comment]
@mergify queue

**Comment by Mic92** [comment]
@mergify refresh

**Comment by Mic92** [comment]
@mergify rebase

**Comment by Mic92** [comment]
@mergify queue

**Pull request merged**